### ISSUE TITLE
Bug when using eloquent Strict mode

### ISSUE THREAD

**Issue by Sergiobop** [itsgoingd/clockwork#604]
Now in Laravel 9.35 you can use shouldBeStrict(), which enables:

🙅‍♂️ No lazy loading

❗ Exceptions when assigning non-fillable attributes

🕵️ Exceptions accessing attributes that weren't retrieved or don't exist

The problem now, it's that clockwork uses the method getKey():
`itsgoingd/clockwork/Clockwork/DataSource/EloquentDataSource.php(201): Illuminate\\Database\\Eloquent\\Model->getKey()`

Now all the queries without the attribute 'id' selected fail. For example:

```
MyModel::query()
      ->select(['name', 'observations'])
      ->get();
```
This query fails:
```
The attribute [id] either does not exist or was not retrieved for model [App\Models\MyModel]
```

**Comment by innocenzi** [comment]
A PR has been merged that should solve this specific use case: https://github.com/laravel/framework/pull/44627

I was confronted to this same issue when Clockwork tried to look into the currently authenticated user.

**Comment by itsgoingd** [comment]
Eloquent strict mode seems like a great way to break all kinds of third-party packages esp. when not selecting id. :)

Anyway, I've released Clockwork 5.1.9 that should work with strict mode.

**Comment by innocenzi** [comment]
Thanks for being reactive about this issue ❤️
